### PR TITLE
Synchronize preproc for pressure and model levels

### PR DESCRIPTION
**Pull request by Peter9192**
Started an attempt to sync preprocessing for model- and pressure levels. Work in progress.

In the process, I moved all unit changes to the tracking (insofar that was not done yet).

Notes:

- pressure levels are descending, model levels ascending?
- level dimension is called "lev" in one file and "level" in the other
- copied total column water correction so it is the same in both files

### CODE REVIEW

**Comment by ruudvdent** [comment]
will conflict with [ #109

**Comment by Peter9192** [comment]
> will conflict with [ #109

I copied all edits from #109 over to this PR.

File: wam2layers/preprocessing/preprocess_era5.py [hunk 1]
```diff
@@ -42,51 +45,40 @@ def load_data(variable, date):
...
     u_surf = load_data("u10", date)  # in m/s
     v_surf = load_data("v10", date)  # in m/s
-    q_surf = calculate_humidity(d2m, p_surf)  # kg kg-1
-
-    # Get grid info
-    time = u.time.values
-    lat = u.latitude.values
-    lon = u.longitude.values
-    a_gridcell, l_ew_gridcell, l_mid_gridcell = get_grid_info(u)
-
```
Comment (ruudvdent) [inline wam2layers/preprocessing/preprocess_era5.py:69]: check for myself (Ruud) when doing the run, if this is indeed not used anymore

**Review by ruudvdent — APPROVED** [review]
looks good

**Pull request merged**